CHANGELOG.md

```diff
@@ -1,23 +1,30 @@
 # Unreleased
- 
+
+# 9.15.0 (Feature release)
+
 ## New features
 
-### Replace back link placeholder URLs with javascript
+### Replace back link placeholder URLs with JavaScript
+
+We've added JavaScript to make the [back link component](https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default, rather than you having to update placeholder text.
+
+If you cannot rely on JavaScript always being available for production scenarios, do not use this JavaScript-reliant approach.
```

Comment:
Sorry @PippaClarkGDS, my bad - I think we might need to tweak this sentence. As @vanitabarrett pointed out, we know people should not rely on JavaScript being available in production, as some users will turn it off or disable it. So, it's not really an if'.

This feature helps people who are quickly building prototypes (which might not be 100% accessible). We want to advise against them simply re-using this code that isn't suitable for production scenarios.

Could the following work?

```suggestion
Do not use this JavaScript-reliant approach in production, as it excludes users who may not have JavaScript available
```

Reply:
@vanitabarrett Just chatted with @PippaClarkGDS about this. How's this wording?

```suggestion
If you want to use JavaScript in production, you must also use a non-JavaScript alternative, so you do not exclude any users.
```

Reply:
@PippaClarkGDS As agreed, I'll add in this wording:

```suggestion
If you want to use JavaScript in production, you must also use a non-JavaScript alternative, so you do not exclude any users. If you cannot use a non-JavaScript alternative, you should hide the back link when JavaScript is not available.
```